rotkehlchen/api/rest.py

```diff
@@ -2597,7 +2597,8 @@ def get_ethereum_airdrops(self, async_query: bool) -> Response:
 
     def get_web3_nodes(self, blockchain: SupportedBlockchain) -> Response:
         nodes = self.rotkehlchen.data.db.get_web3_nodes(blockchain=blockchain)
-        result_dict = _wrap_in_ok_result(process_result_list(list(nodes)))
+        nodes.sort(key=lambda node: node.node_info.name)
```

Comment:
I think it'll be better to the sorting at the DB level. Take a look at
 https://github.com/rotki/rotki/blob/bb6e711fe61690e6b9ba2cd92118b779c7ecfd11/rotkehlchen/db/dbhandler.py#L3315

Reply:
Ha,
we can do that way also. Thanks for pointing out.

Reply:
There can some ways we can present the user the sorted eth rpc nodes in ui :
1. Sort on the client side ( better if list size is small, if size is big then this becomes burden for client. )
2. Get the nodes from db with normal query, sort them on server and present to client via api
3. Get sorted nodes from db query (sorting happens in db servers) and then present to client via api (which you have told above)
4. Maintain the nodes in sorted order in the db, so that we always get nodes already sorted from db. No extra effort. But if there are many updates(add/delete) nodes then it might become costly.
We need to choose best suitable for us.

Reply:
you just need to return them sorted from the DB. It's just a modification on the sql query posted above

Reply:
Ok.
Will do that.

Reply:
pushed changes for that.